README.md

```diff
@@ -3,12 +3,12 @@
 # OpenMRS ESM Template App
 
 This repository provides a starting point for creating your own
-[OpenMRS Microfrontend](https://wiki.openmrs.org/display/projects/OpenMRS+3.0%3A+A+Frontend+Framework+that+enables+collaboration+and+better+User+Experience).
+[OpenMRS Microfrontend](https://openmrs.atlassian.net/wiki/spaces/projects/pages/26936899).
```

Comment:
This link doesn't take me to the right place.

Reply:
@denniskigen what part of the documentation should it lead to?
Is it this https://openmrs.atlassian.net/wiki/x/rIIBCQ

Reply:
Yes